resources/definitions/leapfrog_bolt_pro.def.json

```diff
@@ -0,0 +1,131 @@
+ {
+    "version": 2,
+    "name": "Leapfrog Bolt Pro",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Karan and Vincent 20191022",
+        "manufacturer": "Leapfrog B.V.",
+        "category": "Other",
+        "platform": "leapfrog_bolt_pro_platform.stl",
+        "platform_offset": [0, 0, -14],
+        "file_formats": "text/x-gcode",
+        "supports_usb_connection": false,
+        "supports_network_connection": false,
+        "has_materials": true,
+        "preferred_material": "leapfrog",
+        "has_machine_quality": true,
+        "has_variants": true,
+        "preferred_variant_name": "Brass 0.4",
+        "preferred_material": "leapfrog_epla_natural",
+        "variants_name": "Hot end",
+        "exclude_materials": [
+            "generic_pla",

```

Comment:
You don't need to mention all the materials that you couldn't use in the first place (eg; Your machine is 1.75 mm, so you don't need to exclude generic_pla, since that is a 2.85mm material and will be filtered away because of that)

Reply:
Thank you for your feedback. We have removed all materials that are being filtered away because of a different diameter.